app/src/main/java/com/movies/data/remote/di/NetworkModule.kt

```diff
@@ -0,0 +1,43 @@
+package com.movies.data.remote.di
+
+import com.movies.data.remote.service.api.ServiceApi
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import org.koin.dsl.module
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+
+private const val BASE_URL = "https://api.themoviedb.org/"
+
+private fun createInterceptor(): Interceptor {
+    return Interceptor { chain ->
+        val request = chain.request().newBuilder()
+            .header("Authorization", ApiKey.VALUE)
+            .build()
+        chain.proceed(request)
+    }
+}
+
+private fun creteOkhttpClient(): OkHttpClient {
+    return OkHttpClient.Builder()
+        .addInterceptor(createInterceptor())
+        .build()
+}
+
+private fun createRetrofit(): Retrofit {
+    return Retrofit.Builder().baseUrl(BASE_URL)
+        .addConverterFactory(GsonConverterFactory.create())
+        .client(creteOkhttpClient())
+        .build()
+}
+
+val networkModule = module {
+    single { createRetrofit() }
+    single { get<Retrofit>().create(ServiceApi::class.java) }
+}
+
+object ApiKey {
```

Comment:
Extract to network constants. Should not be in di

Reply:
done